ocs_ci/utility/performance_dashboard.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+####################################################
+# Sample script to save result data using json #
+####################################################
+import requests
+import json
+from datetime import datetime
+from ocs_ci.framework import config
+from ocs_ci.ocs import ocp, defaults
+from ocs_ci.ocs.node import get_typed_nodes
+from ocs_ci.ocs import constants
+
+current_date = datetime.today()
+URL = 'http://10.0.78.167:8000/'
+
+
+def push_perf_dashboard(
+    interface, read_iops, write_iops, bw_read, bw_write
+):
+    """
+
+    """
+    csv = ocp.OCP(kind='csv', namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+    worker_type = get_typed_nodes(num_of_nodes=1)[0].data['metadata'][
+        'labels'
+    ]['beta.kubernetes.io/instance-type']
+    interface = (
+        constants.RBD_INTERFACE if interface == constants.CEPHBLOCKPOOL else (
+            constants.CEPHFS_INTERFACE
+        )
+    )
+    csv_vers = csv.get()['items'][1]['spec']['version'][:-3].split("-")
+    min_version = csv_vers[0]
+    build_id = csv_vers[1]
+    ocs_ver = ".".join(min_version.split(".")[:-1])
+    platform = config.ENV_DATA['platform']
+    if platform.lower() == 'aws':
+        platform = platform.upper() + " " + worker_type
+    sample_data = [
```

Comment:
We have to consider that this same json can be changed depends on the test  we running and parameters we push to server

Reply:
every usage of this push has to be the same.
If a performance test uses this function it has to pass:
interface, read_iops, write_iops, bw_read, bw_write